lib/id3c/cli/command/etl/redcap_det.py

```diff
@@ -63,6 +62,39 @@ def get_redcap_record(project_id: str, record_id: str, token_name: str) -> dict:
     return r.json()[0]
 
 
+class InstrumentStatus(Enum):
+    """
+    Numeric and string codes used by REDCap for instrument status.
+    """
+    Incomplete = 0
+    Unverified = 1
+    Complete = 2
+
+
+def is_complete(instrument: str, record: dict) -> bool:
+    """
+    Test if the named *instrument* is marked complete in the given *record*.
+
+    >>> is_complete("test", {"test_complete": "Complete"})
+    True
+    >>> is_complete("test", {"test_complete": 2})
+    True
+    >>> is_complete("test", {"test_complete": "2"})
+    True
+    >>> is_complete("test", {"test_complete": "Incomplete"})
+    False
+    >>> is_complete("test", {})
+    Traceback (most recent call last):
+        ...
+    KeyError: 'test_complete'
+    """
+    return record[f"{instrument}_complete"] in {
+        InstrumentStatus.Complete.name,
+        InstrumentStatus.Complete.value,
+        str(InstrumentStatus.Complete.value)
+    }
```

Comment:
@tsibley why did you put the complete statuses in a set instead of a list?

Reply:
It's a set. There are two ways to make a set in Python:

1. The `set()` constructor which takes an iterable.
2. Set literal syntax, which is `{ value, value, value, … }`. It is similar to dict literals (`{ key: value, key: value, … }`) but omits the `key:` part.

Set literals are used in various places around ID3C.

Reply:
I thought it was a set but then I doubted myself :) Thanks!